src/components/_global/BalAsset/BalAssetSet.vue

```diff
@@ -43,23 +50,58 @@ export default defineComponent({
     size: {
       type: Number,
       default: 26
+    },
+    maxAssetsPerLine: {
+      type: Number,
+      default: 8
     }
   },
 
   setup(props) {
-    function leftOffsetFor(i: number): number {
-      const maxCount = 8;
-      const radius = props.size / 2;
-      const spacer = (maxCount / props.addresses.length - 1) * (radius * 2);
-      return ((props.width - radius * 2 + spacer) / (maxCount - 1)) * i;
```

Comment:
I extracted some of the calculations into computables since they don't really change for a given `i`.